runtime/onert/core/src/ir/LayoutSet.test.cc

```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  */
 
-#include <gtest/gtest.h>
+#include "LayoutSet.h"
```

Comment:
@hseok-oh (out of curiosity) Why it is not `ir/LayoutSet.h` like others?

Reply:
`LayoutSet.h` is in src/ (private header), not include/ (public header).